package.json

```diff
@@ -118,5 +121,15 @@
     "test:unit": "mocha test/unit",
     "test:server": "mocha test/server",
     "test--browser": "browserify test/frontend/frontend.bundle.js -o test/frontend/bundle.js -d && node test/frontend/driver.js"
+  },
+  "lint-staged": {
+    "*.js": [
+      "prettier --single-quote --write",
+      "eslint",
+      "git add"
+    ],
+    "*.css": [
+      "stylelint"
```

Comment:
nit: Not sure if we want to run `prettier --single-quote --write` here too or not. If so, we'd probably also need a `git add` step in case **prettier** rewrites the CSS.